prow/plugins/label/label.go

```diff
@@ -226,19 +230,26 @@ func handle(gc githubClient, log *logrus.Entry, ae assignEvent) error {
 	}
 
 	toRepeat := []string{}
+	isMember := false
 	if len(sigMatches) > 0 {
-		isMember, _ := gc.IsMember(ae.org, ae.login)
-		if !isMember {
-			toRepeat = ae.getRepeats(sigMatches, existingLabels)
-		}
+		isMember, _ = gc.IsMember(ae.org, ae.login)
+		toRepeat = ae.getRepeats(sigMatches, existingLabels)
 	}
 
-	if len(toRepeat) > 0 {
+	if (len(toRepeat) > 0) || !isMember {
 		msg := fmt.Sprintf(chatBack, strings.Join(toRepeat, ", "))
 		if err := gc.CreateComment(ae.org, ae.repo, ae.number, plugins.FormatResponseRaw(ae.body, ae.url, ae.login, msg)); err != nil {
 			log.WithError(err).Errorf("Could not create comment \"%s\".", msg)
 		}
 
+		if sc != nil {
+			//if sig matches then send a notification on slack
```

Comment:
I have a semantic issue with a plugin called "label" doing something slack-related.  Could we pull out into a separate plugin?